Cargo.toml

```diff
@@ -22,4 +22,4 @@ url = "2.0"
 
 [dev-dependencies]
 dotenv = "0.15"
-tokio = { version = "0.2", features = ["macros", "rt-core", "test-util"] }
+tokio = { version = "1.3.0", features = ["full"] }
```

Comment:
Why did you have to pull down all of the features?

Reply:
sure! let me have a deep in. will back to you later